<launch>
  <!-- All Rosaria nodes -->
  <node pkg="rosaria" type="RosAria" name="RosAria0" output="screen"/>

  <!-- All Get_Coord nodes (only if sim=False) -->

  <!-- The Sensor nodes -->
  <node pkg="controller" type="PoseHandler" name="Sensor0" args="0" output="screen"/>

  <!-- All Filter nodes -->
  <node pkg="controller" type="KalmanFilter" name="Filter0" args="0"  output="screen"/>

  <!-- All Robot nodes -->
  <node pkg="controller" type="RobotHandler" name="Robot0" args="0" output="screen"/>

  <!-- The Controller node -->
  <node pkg="controller" type="PresentationProblem" name="Controller" output="screen"/>

</launch>
